### DISCUSSION TITLE
How to change the size of the Chairman's comments box in the chairman's worksheet

### DISCUSSION THREAD

**Discussion by ajtruckle** [ajtruckle/meeting-schedule-assistant-support#45] [Q&A] [answered]
I received this question via email:

> Re the Chairman’s worksheet I just wondered if I could make the box under the Chairman’s opening comments and closing comments a bit smaller?
> 
> I’m having to shrink the size of the whole thing to fit on one side of A4 then the size of the font also reduces…
> 
> It seems with the new meeting format there’s more description about the assignments, (this extra text is excellent when you are chairman so I need to keep that!)
> 
> I’ve worked out that removing the odd <br />  reduces the blank space under each assignment but I just can’t workout what to change to make the two other chairman’s boxes a bit smaller…

**Comment by ajtruckle** [comment, marked as answer]
It is always a good idea to view the schedule in an external browser and then using the Inspect tool to see how things are working. An external browser session can be started from within the Meeting Editor, eg:

File > Preview in Browser > Microsoft Edge

If you right-click the white area of the opening comment and choose Inspect from the context menu you will see:

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/7985331/d8c22b65-3104-499d-b45c-6d46a4fe232f)

Upon inspection you can see how the HTML and CSS looks. Can you see this CSS:

```css
.textChairmanNotes {
    font-size: 10pt;
    padding: 2mm;
    margin: 0;
    min-height: 25mm;
    text-align: left;
}
```

Notice that it is using a minimum height of 25mm (`min-height: 25mm;`)?  Reducing this value will reduce the minimum height of the chairman notes.

Keep in mind that you should always work on a copy of the template (XSL) and style sheet (CSS) files. It is assumed that you already know how to do this. Simply locate the `textChairmanNotes` class in your CSS file and reduce the `min-height` value as desired.
<hr>
Now, what if you wanted to make changes to the template? In this case you can open your template and locate this section:

```xsl
<xsl:template match ="OpeningComments | ClosingComments | Treasures1 | Treasures2 | Living1 | Living2 | Living3" mode="ChairmanNotes">
	<tr>
		<td colspan="4" class="cellChairmanNotes">
			<div class="textChairmanNotes">
				<xsl:choose>
					<xsl:when test="self::OpeningComments">
						<xsl:value-of select="Comments" disable-output-escaping="yes"/>
					</xsl:when>
					<xsl:when test="self::ClosingComments">
						<xsl:value-of select="Comments" disable-output-escaping="yes"/>
						<xsl:apply-templates select="ClosingAnnouncements"/>
						<xsl:if test="@IncludeNextWeeksStudents='1'">
							<xsl:variable name="AssignHistoryPath">
								<xsl:choose>
									<xsl:when test="//Settings/ForeignGroupMode=1">ForeignAssignHistory.xml</xsl:when>
									<xsl:otherwise>AssignHistory.xml</xsl:otherwise>
								</xsl:choose>
							</xsl:variable>
							<xsl:variable name="AssignHistory" select="document($AssignHistoryPath)"/>
							<xsl:variable name="week" select="../../Date/@NextWeek"/>
							<div class="NextWeeksStudents-Container">
								<xsl:apply-templates select="$AssignHistory/AssignmentHistory/*[name()=$week]/StudentItems">
									<xsl:with-param name="MainHall" select="//Labels/MainHall"/>
									<xsl:with-param name="AuxClass1" select="//Labels/AuxClass1"/>
									<xsl:with-param name="AuxClass2" select="//Labels/AuxClass2"/>
								</xsl:apply-templates>
							</div>
						</xsl:if>
					</xsl:when>
					<xsl:otherwise>
						<xsl:value-of select="." disable-output-escaping="yes"/>
					</xsl:otherwise>
				</xsl:choose>
			</div>
		</td>
	</tr>
</xsl:template>
```
Structural changes can be made in that part of the template if needed. But it should be sufficient to reduce the `min-height` in the CSS file for your issue.
<hr>
Please be aware that you can also use the Inspect feature within the Meeting Editor but you might have to click Inspect twice on the context menu for it to drill dow to the element of interest.

**Reply by srv-999** [reply]
as always a simple and complete explanation - thanks Andrew - I'm using 15mm now and it seems to be ok.
